package.json

```diff
@@ -73,6 +73,8 @@
     "pathval": "~0.1.1",
     "prompt": "~0.2.14",
     "strip-json-comments": "~1.0.2",
+    "to-double-quotes": "^1.0.0",
```

Comment:
The majority of our dependencies use tildes. What's the heuristic to determine if we should use caret or tilde?